Makefile

```diff
@@ -310,7 +310,7 @@ format-python:
 	cd ${ROOT_DIR}/sdk/python; python -m black --target-version py38 feast tests
 
 lint-python:
-	cd ${ROOT_DIR}/sdk/python; python -m mypy
+	cd ${ROOT_DIR}/sdk/python; python -m mypy --exclude=/tests/ --follow-imports=skip feast
```

Comment:
I would feel this is the biggest change, is that I only limit the linting checks to sdk/python/feast directory

Reply:
what if we still check tests? a lot of changes needed right?